app/classes/query/initializers/names.rb

```diff
@@ -32,19 +32,45 @@ def names_parameters
       end
 
       def initialize_name_parameters(*joins)
-        table = params[:include_nonconsensus] ? "namings" : "observations"
-        column = "#{table}.name_id"
-        add_id_condition(column, lookup_names_by_name(names_parameters), *joins)
-        add_join(:observations, :namings) if params[:include_nonconsensus]
-        return unless params[:exclude_consensus]
+        if irreconcilable_name_parameters?
```

Comment:
Good idea about the guard clause. I'll do it this way:
```ruby
        return add_false_condition if irreconcilable_name_parameters?
```
and get rid of force_empty_results_xxx:
- I'd prefer to avoid something like handle_irreconcilable_name_parameters because it has a dual purpose: change state and return t/f.
- `add_false_condition` parallels `add_id_condition` and `add_not_id_condition`